app/uk/gov/hmrc/eoricommoncomponent/frontend/connector/UpdateVerifiedEmailConnector.scala

```diff
@@ -18,38 +18,24 @@ package uk.gov.hmrc.eoricommoncomponent.frontend.connector
 
 import play.api.Logger
 import play.mvc.Http.Status._
-import uk.gov.hmrc.eoricommoncomponent.frontend.audit.Auditable
 import uk.gov.hmrc.eoricommoncomponent.frontend.config.AppConfig
-import uk.gov.hmrc.eoricommoncomponent.frontend.connector.httpparsers.{
-  BadRequest,
-  Forbidden,
-  HttpErrorResponse,
-  ServiceUnavailable,
-  UnhandledException,
-  VerifiedEmailRequest,
-  VerifiedEmailResponse
-}
+import uk.gov.hmrc.eoricommoncomponent.frontend.connector.httpparsers._
 import uk.gov.hmrc.http.{ForbiddenException, HttpClient, _}
 
 import javax.inject.Inject
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 import scala.util.control.NonFatal
 
-class UpdateVerifiedEmailConnector @Inject() (appConfig: AppConfig, http: HttpClient, audit: Auditable) {
+class UpdateVerifiedEmailConnector @Inject() (appConfig: AppConfig, http: HttpClient) {
 
   private val url: String = appConfig.getServiceUrl("update-verified-email")
   private val logger      = Logger(this.getClass)
 
-  def updateVerifiedEmail(request: VerifiedEmailRequest, currentEmail: Option[String])(implicit
-    hc: HeaderCarrier
-  ): Future[Either[HttpErrorResponse, VerifiedEmailResponse]] = {
-    val newEmail = request.updateVerifiedEmailRequest.requestDetail.emailAddress
-    val eori     = request.updateVerifiedEmailRequest.requestDetail.IDNumber
-
-    auditRequest(currentEmail, newEmail, eori, "changeEmailAddressVerified")
+  def updateVerifiedEmail(
+    request: VerifiedEmailRequest
+  )(implicit hc: HeaderCarrier): Future[Either[HttpErrorResponse, VerifiedEmailResponse]] =
     http.PUT[VerifiedEmailRequest, VerifiedEmailResponse](url, request) map { resp =>
-      auditRequest(currentEmail, newEmail, eori, "changeEmailAddressConfirmed")
```

Comment:
Moved this as well as I think it only makes sense to execute this audit event when we receive correct payload indicating that email was verified (non 003 or any other error response).

Reply:
Also fine with this one.